automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala

```diff
@@ -280,6 +282,64 @@ trait Orchestration extends FireCloudClient with LazyLogging {
     postRequestWithMultipart(apiUrl(s"api/workspaces/$ns/$wsName/importEntities"), fileName, fileContent)
   }
 
+  object trial {
+
+    case class TrialProjectReport(unverified: Int,
+                                  errored: Int,
+                                  available: Int,
+                                  claimed: Int)
+
+    def enableUser(userEmail: String): Unit = {
+      val campaignManager = UserPool.chooseCampaignManager
+      implicit val token: AuthToken = campaignManager.makeAuthToken()
+      logger.info(s"Enabling user [$userEmail] using campaign manager [${campaignManager.email}]")
+      val enableResponse: String = postRequest(apiUrl("api/trial/manager/enable"), Seq(userEmail))
+      val responseJson: JsObject = enableResponse.parseJson.asJsObject
+      val successfulResponseKeys = Seq("Success", "NoChangeRequired")
+      responseJson.fields.map {
+        case f@x if successfulResponseKeys.contains(f._1) =>
+          logger.info(s"${f._1}: ${f._2.toString()}")
+          return
+        case f@y =>
+          logger.error(s"${f._1}: ${f._2.toString()}")
+          throw new Exception(s"Unable to enable user: $userEmail. Error message: $enableResponse")
+      }
+    }
+
```

Comment:
I don't think `createTrialProjects` would work the way we want for the case where, say, there's one project available and we want to create five projects. Do we want to replace

```scala
if (report.available == 0) {
```
by
```scala
if (report.available < count) {
```
and
```scala
postRequest(apiUrl(s"api/trial/manager/projects?operation=create&count=$count"))
```
by
```scala
postRequest(apiUrl(s"api/trial/manager/projects?operation=create&count=${count - reports.available}"))
```
?